tests/test_kb_readonly.py

```
@@ -0,0 +1,428 @@
+    _kb_answer_status,
+)
+
+
+# ---------------------------------------------------------------------------
+# Helpers
+# ---------------------------------------------------------------------------
+
+
+def _make_response(
+    status_code: int = 200,
+    json_body: Any = None,
```

Ruff (0.15.12)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `json_body`

(ANN401)